linters/badtime/README.md

```diff
@@ -0,0 +1,62 @@
+# Badtime
```

Comment:
I'm not sure why its showing so many changes to the readme, I just replaced maptime with badtime and added a few more sentences explaining the expanded functionality

Reply:
Oh its because I renamed the file, thats why